CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ changelog
 =========
 This changelog follows the patterns described here: https://keepachangelog.com/en/1.0.0/.
 
-Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, sexurity`.
```

Comment:
Ahh man! This one is good. We should just leave it 🤣

Reply:
I agree, it should be left in there. Maybe an easter-egg in documentation?